Generation/doc/AddingPileupToGeneration.md

```diff
@@ -0,0 +1,33 @@
+
+
+# Adding Pileup in FCCSW in Generation
+
+Some of the FCC scenarios have a large number of pileup collisions in each event.
+A full simulation of the full event is very expensive, but some studies may nevertheless require the full information.
+For this purpose, FCCSW includes infrastructure to add pileup at generator level.
+
+
+By default, the particle sources in the Generator package of FCCSW can handle pileup, but are set to a default of no pileup.
+The standard way of adding pileup is to add a `Pileup` tool, which has a and configure the number of pileup. There are different implemenations of this tool in FCCSW, depending on the desired distribution (Constant or Poisson-distributed number of pileup events mixed in each event)
+
+
+## Pileup parameters
+
+As a convenient access to FCC-hh pileup parameters, FCCSW includes some small Python dictionaries with the relevant values in `Generation/python/FCCPileupScenarios.py` which can be used in configuration files by, p.ex.,
```

Comment:
You may drop phrases "as a convenient access...". We should explain things, not advertise ;-)